apps/frontend/static_src/scss/components/doc-mode.scss

```diff
@@ -0,0 +1,17 @@
+.doc-mode:hover {
+    .doc-mode__container {
+        .doc-mode__row1 {
```

Comment:
line 2-3 do not add styling, we remove them. Same for the HTML,if we don't apply styling, there shoudn't be class names.